src/main/java/com/icthh/xm/ms/mstemplate/config/AsyncConfiguration.java

```diff
@@ -0,0 +1,48 @@
+package com.icthh.xm.ms.mstemplate.config;
+
+import java.util.concurrent.Executor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.aop.interceptor.AsyncUncaughtExceptionHandler;
+import org.springframework.aop.interceptor.SimpleAsyncUncaughtExceptionHandler;
+import org.springframework.boot.autoconfigure.task.TaskExecutionProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.scheduling.annotation.AsyncConfigurer;
+import org.springframework.scheduling.annotation.EnableAsync;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor;
+import tech.jhipster.async.ExceptionHandlingAsyncTaskExecutor;
+
+@Configuration
+@EnableAsync
+@EnableScheduling
+@Profile("!testdev & !testprod")
+public class AsyncConfiguration implements AsyncConfigurer {
```

Comment:
Please check if we have the same or similar config in commons. 
In not - maybe it make sense to move such config to commons lib to avoid duplication when MS template is applied